### PR TITLE
Fix metavariables in rtt.sub spec

### PR DESCRIPTION
**Pull request by tlively**

### CODE REVIEW

File: proposals/gc/MVP.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ Note: The `br_on_*` instructions allow an operand of unrelated reference type, e
 
 * `rtt.sub <typeidx>` returns an RTT for `typeidx` as a sub-RTT of a the parent RTT operand
   - `rtt.sub $t : [(rtt n $t')] -> [(rtt (n+1) $t)]`
-    - iff `t2 <: t1`
+    - iff `t <: t'`
```
Comment (tlively) [inline proposals/gc/MVP.md:379]: Should there be an additional requirement that `t != t'`, or is intended that the same type can be used in successive rtts in a chain?

Comment (kripken) [reply proposals/gc/MVP.md:379]: A possible use of `t == t'` is that `(ref.canon $foo) != (ref.sub $foo (ref.canon $foo))`. But I do not know if that is useful in practice.

Comment (RossTate) [reply proposals/gc/MVP.md:379]: If rtt depth is supposed to be able to correspond to class-inheritance depth, then `t == t'` would happen when a subclass does not add new fields.

Comment (rossberg) [reply proposals/gc/MVP.md:379]: Ah, thanks, but the symbolic names are merely indices now. So I think it needs to say this:
```suggestion
    - iff `(type $t) <: (type $t')`
```

**Pull request merged**